README.md

```diff
@@ -58,18 +47,21 @@ Decentralized websites using Bitcoin crypto and the BitTorrent network - https:/
 
 #### [More screenshots in ZeroNet docs »](https://zeronet.readthedocs.org/en/latest/using_zeronet/sample_sites/)
 
+Do you want visit sites with .bit extension without localhost address? You can download this extension:
```

Comment:
should be: "Do you want __to__ visit"

Reply:
https://chrome.google.com/webstore/detail/zeronet-protocol/cpkpdcdljfbnepgfejplkhdnopniieop doesn't exist anymore.